ietf-dtnma-agent.yang

```diff
@@ -452,6 +452,81 @@ module ietf-dtnma-agent {
     }
   }
 
+  // Objects related to ODM handling
+  amm:edd odm-list {
+    amm:enum 16;
+    description
+      "A table of ODM within the agent.";
+    uses obj-list-params;
+    amm:tblt {
+      amm:key "org-name odm-name";
```

Comment:
Could add an `amm:unique "org-enum odm-enum";` as well but there should be normative constraints described in the description to expect when this will work and when will fail.